modules/persistenceService/src/main/scala/connectionLayer/DbQueries.scala

```diff
@@ -0,0 +1,23 @@
+package connectionLayer
+
+import persistenceAlgebra.DbAlgebra
+import persistenceModel.{User, UserName}
+import doobie._
+import doobie.implicits._
+import DbDefinitions._
+import cats.effect.Bracket
+
+object DbDefinitions {
+  val tableName = ""
+}
+
+class DbQueries[F[_]](implicit ev: Bracket[F, Throwable]) extends DbAlgebra {
+  def insert(user: User): ConnectionIO[Int] =
+    sql"insert into $tableName(name) values (${user.name})".update.run
```

Comment:
Since this is tied to the user, I would say it's fine to explicitly specify the table name here
```suggestion
    sql"insert into user values (${user.name})".update.run
```

Same for the others